cf-java-logging-support-core/src/main/java/com/sap/hcp/cf/logging/common/request/RequestRecordBuilder.java

```diff
@@ -39,7 +39,7 @@ public RequestRecordBuilder addOptionalTag(boolean optionalFieldCanBeLogged, Str
 			requestRecord.addTag(fieldKey, Defaults.REDACTED);
 		}
 
-		if (!optionalFieldCanBeLogged && tag.equals(Defaults.UNKNOWN)) {
+        if (!optionalFieldCanBeLogged && Defaults.UNKNOWN.equals(tag)) {
```

Comment:
```suggestion
		if (!optionalFieldCanBeLogged && Defaults.UNKNOWN.equals(tag)) {
```

Reply:
fix indentation

Reply:
I reformatted the whole class with the correct formatting rules. It was once accidentally formatted wrongly by me.